web-server/v0.4/.ga

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why do we need this `.ga` file?

Reply:
This is not needed and will be removed under this PR.